packages/manager/src/components/user/LoginPage.tsx

```diff
@@ -52,22 +77,47 @@ export default () => {
   return (
     <Page header={<Header />}>
       <PageSection variant={PageSectionVariants.light} isFilled>
-        <Bullseye>
-          <Card className={css(styles.card)}>
-            <CardHeader>
-              <Title headingLevel="h6" size="md">
-                Sign in with
-              </Title>
-            </CardHeader>
-            <CardBody>
-              <Button isBlock variant="primary" onClick={onClickLogin} className="spaship_btn">
-                Red Hat SSO
-              </Button>
-            </CardBody>
-          </Card>
-        </Bullseye>
+        <div className={css(styles.defcard)}>
+          <h5 className={css(styles.h5)}>
+            develop fast · <span className={css(styles.yellow)}>deploy faster</span>
+          </h5>
+          <div className={css(styles.definition)}>
+            SPAship is a open source platform for deploying, integrating, and managing single-page apps (SPAs).
+          </div>
+        </div>
+        <Card className={css(styles.card)}>
+          <CardHeader>
+            <Title headingLevel="h6" size="md">
+              Sign in with
+            </Title>
+          </CardHeader>
+          <CardBody>
+            <Button isBlock variant="primary" onClick={onClickLogin} className="spaship_btn">
+              Red Hat SSO
+            </Button>
+          </CardBody>
+        </Card>
+      </PageSection>
+      <PageSection
+        className={css(styles.footer)}
+        variant={PageSectionVariants.darker}
+        style={{ height: "6em" }}
+        isFilled={false}
+      >
+        Brought to you by the{" "}
+        <a href="https://github.com/spaship/spaship/graphs/contributors" target="_">
```

Comment:
The `target` should be `_blank`,  the `_` is invalid in target attribute .

and `rel="noreferrer noopener"` is recommend if you are use `target`
ref: [<a>: The Anchor element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a)

```suggestion
        <a href="https://github.com/spaship/spaship/graphs/contributors" target="_blank" rel="noreferrer noopener">
```

Reply:
Done.